tests/TokenizerTest.php

```diff
@@ -7,9 +7,44 @@
 use Doctrine\SqlFormatter\Tokenizer;
 use PHPUnit\Framework\Attributes\DoesNotPerformAssertions;
 use PHPUnit\Framework\TestCase;
+use ReflectionClass;
+
+use function sort;
 
 final class TokenizerTest extends TestCase
 {
+    /**
+     * @param 'reserved'|'reservedToplevel'|'reservedNewline'|'functions' $propertyName
+     *
+     * @return list<string>
+     */
+    private function getTokenizerList(string $propertyName): array
+    {
+        $tokenizerReflClass = new ReflectionClass(Tokenizer::class);
+        /** @var list<string> $res */
+        $res = $tokenizerReflClass->getProperty($propertyName)->getDefaultValue();
+
+        return $res;
+    }
+
+    public function testInternalKeywordListsAreSortedForEasierMaintenance(): void
```

Comment:
i think that this should be a code style check rather than an unit test.

Reply:
If you know how, then great and definitely tell me, if not, let's move with this PR. In #106 I need to add many keywords and I want to make sure the git diff is minimal for now and long term.